zulipterminal/config/keys.py

```diff
@@ -461,6 +467,25 @@ def commands_for_random_tips() -> List[KeyBinding]:
     ]
 
 
+def display_keys_for_command(command: str) -> List[str]:
+    try:
+        urwid_keys = KEY_BINDINGS[command]["keys"]
+        return display_keys_for_urwid_keys(urwid_keys)
+    except KeyError as exception:
+        raise InvalidCommand(command) from exception
+
+
+def display_keys_for_urwid_keys(urwid_keys: List[str]) -> List[str]:
+    try:
+        return [URWID_KEY_TO_DISPLAY_KEY_MAPPING[key] for key in urwid_keys]
```

Comment:
Using this new lookup table does work, but we want to avoid needing to maintain this table except for specific keys that we want to render differently.

I think you had a previous version which checked if it was in a lookup table (dict) like this and then used the key if not? That would be sufficient for now.

Later if we want the single-key capitalization, we should use a different approach too, since storing a mapping of letter->capital is not worth it - this can be done algorithmically.